_pages/en/capabilities.md

```diff
@@ -4,47 +4,218 @@ title: High Performing IT Capabilities
+
+#### 1.2 Continuous integration
+
+##### 1.2.1 Outcomes
+
+Teams must integrate all their work into the main version of the code base on a regular basis.
+Changes must trigger automated builds of the software and a series of automated tests that provide feedback in a few minutes.
+Builds should be numbered and repeatable.
+
+Tests should run successfully at least once a day.
+The system should also make the status visible to the team.
+
+##### 1.2.2. Indicators
+
+- What percentage of code commits result in a software build without manual intervention?
+- What percentage of code commits result in a suite of automated tests being run without manual intervention?
+- What percentage of automated builds and tests that are executed successfully every day?
+- Are builds available to testers?
+- What percentage of tests provide feedback that is available to developers within a day?
+- How long does it takes between the build breaking and having it fixed, either with a check-in that fixes the problem, or by reverting the breaking change?
+- Are builds numbered and repeatable?
+- Are automated tests run every day?
+- How long do tests and build take?
+
+How to measure?
+
+##### 1.2.3 Goals
+
+- All code commits result in a software build without manual intervention
+- All code commits result in a suite of automated tests being run without manual intervention
+- 85% of automated builds and tests that are executed successfully every day
+- All tests provide feedback that is available to developers within a day
+- Fixing build or failing tests takes [1-24] hours
+
+##### 1.2.4. How to improve
+
+- Use version control
+- Remove manual build steps - automate
+- Trigger quick tests on every change (unit tests) for quick feedback
+- Make fixing a broken build a priority
+- Limit tests to 10 minutes
+- Merge changes frequently into trunk
+
+### 2. Process Capabilities
+
+#### 2.2 Streamlining change approval
+
+##### 2.2.1 Outcomes
+
+...
+
+##### 2.2.2. Indicators
+
+- What percentage of changes do not require a manual change approval to be promoted to production?
+- what percentage of low-, medium-, and high-risk changes require a manual change approval to be promoted to production?
+- What amount of time do changes spend waiting for approval from external bodies?
+- What percentage of changes are managed by peer-review?
+
+##### 2.2.3 Goals
+
+- 90% of changes do not require a manual change approval to be promoted to production
+- 100% of low-risk changes require a manual change approval to be promoted to production
+- 90% of medium-risk changes require a manual change approval to be promoted to production
+- 75% of high-risk changes require a manual change approval to be promoted to production
+- Changes spend [1-7] days waiting for approval from external bodies
+- 10% of changes are managed by peer-review
+
+### 3. Measurement Capabilities
+
+..
+
+### 4. Cultural capabilities
+
+#### 4.2 Westrum organizational culture
```

Comment:
Reminder that we had a stakeholder engagement plan put on hold. In this plan, we were to interview 36 teams (director level) with one question about assessing the westrum org culture (how does the director's team assess IITB as an organization).

But if we could have this as a self-assessment tool, we could capture how individual team rates themselves (not IITB, but themselves). Removing bias against the team's  management.

So we could have 2 metrics:
1) IITB as a whole (36 interviews)
2) IITB individual teams

Reply:
Good idea, could probably be put together real quick :thinking:

Reply:
@gcharest it was already done (with @RachelMuston , @myp192 , and @ElminaIusifova ).We even did 2 test interviews. The pandemic put the whole initiative on hold

Reply:
The things you miss when you're not there :man_shrugging: 

Go team! :clap: